### ISSUE TITLE
1.4.9 Error logging question

### ISSUE THREAD

**Issue by Bahbv** [evolution-cms/evolution#1227]
Hello,
Is it possible to always show the  "Evo Parse Error"  Error page? Even if the user is not logged in?
I want to be able to check where things went wrong even when i can't access the manager.
The abillity to print errors to an .TXT instead of the database would also be great.

Thanks in advance.

**Comment by Bahbv** [comment]
The only solution i found myself is editing line 6435 in document.parser.class to **not** check if the user is in mgrValidated session. Not clean, but it worked.

        //Display error
        $bDebugPrint = true;
		
        ob_get_clean();
        if (isset($_SESSION['mgrValidated']) || ($bDebugPrint == true)) {
            echo '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"><html><head><title>EVO Content Manager " . $version . " &raquo; " . $release_date . '</title>
                 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
                 <link rel="stylesheet" type="text/css" href="' . $this->config['site_manager_url'] . "media/style/' . $this->config['manager_theme'] . '/style.css" />
                 <style type="text/css">body { padding:10px; } td {font:inherit;}</style>
                 </head><body>
                 " . $str . '</body></html>';

        } else {
            echo "Error';
        }
        ob_end_flush();
        exit;
    }

**Comment by bossloper** [comment]
A bit risky maybe. Could allow hackers to work out a way in to your site?

**Comment by Dmi3yy** [comment]
Yes, is fast solution how to see error(but after fix recommend set to false  ), in Evo 2.0 log write into files too.
